changelog/v1.24.0-patch1/upgrade-envoy.yaml

```diff
@@ -0,0 +1,6 @@
+changelog:
```

Comment:
should we go to 1.24.1 https://www.envoyproxy.io/docs.html ? rather than 1.24.0

Reply:
since 1.24 has been released, i would expect this to be a patch release rather than rc (im assuming the upstream envoy commit is 1.24.0 or newer)

Reply:
istio 1.16.0 is only at 1.24.0. hence not going to 1.24.1